keras_eval/utils.py

```diff
@@ -537,4 +539,7 @@ def results_differential(dataframes, mode='average', round_decimals=4):
     else:
         raise ValueError('Results mode must be either "average" or "individual"')
 
+    if save_csv_path is not None:
```

Comment:
```suggestion
    if not save_csv_path:
```

Although I agree that this short form is not as "easy" to read as the existing.

Reply:
https://stackoverflow.com/questions/3901144/difference-between-if-x-and-if-x-is-not-none